app/_src/guides/progressively-rolling-in-strict-mtls.md

```diff
@@ -175,13 +219,15 @@ flowchart LR
 
 Finally, to set strict mode you can either edit the policy or remove it (the default is taken from `Mesh` object which is `STRICT`).
 
-```bash
-kubectl delete meshtlses.kuma.io -n kuma-system redis
-```
+{% tip %}
+**Things to remember when migrating to strict TLS**
 
-## Things to remember when migrating to strict TLS
+The difference between `cluster.localhost_6379.upstream_cx_total` and `inbound_10_42_0_13_6379.rbac.allowed` before changing a workload to `Strict` mode equals that after changing it.
```

Comment:
This seems more correct to me, but would love input from a native speaker - @michaelbeaumont - can you take a look?

```suggestion
If there is only encrypted traffic sent to the destination the difference between `cluster.localhost_6379.upstream_cx_total` and `inbound_10_42_0_13_6379.rbac.allowed` before changing a workload to `Strict` mode will equal that after changing it.
```

Reply:
My suggestion:

```suggestion
If only encrypted traffic is sent to the destination, the difference between `cluster.localhost_6379.upstream_cx_total` and `inbound_10_42_0_13_6379.rbac.allowed`  will not change after setting the workload to `Strict` mode.
```

"that after changing it" isn't right and "the difference doesn't change" sounds better than "the difference equals .."